package.json

```diff
@@ -11,11 +11,16 @@
   "scripts": {
     "test": "npm-run-all --print-name  --sequential test:make test:format test:examples test:run  test:review ",
     "test:make": "elm make --docs=docs.json",
-    "test:format": "elm-format src/ tests/*.elm tests/String review --validate",
+    "test:format": "elm-format src/ tests/*.elm tests/String review benchmarks --validate",
     "test:examples": "elm-verify-examples",
     "test:run": "elm-test",
     "test:review": "elm-review",
-    "preview-docs": "elm-doc-preview",
+    "fix": "npm-run-all --print-name --sequential fix:format fix:review",
+    "fix:format": "elm-format src/ tests/*.elm tests/String review benchmarks --yes",
+    "fix:review": "elm-review --fix",
+    "start": "npm-run-all start:docs",
+    "start:docs": "elm-doc-preview",
+    "start:bench": "cd benchmarks; elm make --optimize --output elm-stuff/bench.html src/Benchmarks.elm && open elm-stuff/bench.html",
```

Comment:
Personally I think that the benchmarking methodology of elm-benchmark already skews to the unrealistic for most applications due to JIT warming, when most uses of these functions will never kick in to higher optimisation levels.

eol2 as far as I can tell isn't widely deployed in production applications. So if for instance one implementation was better in `--optimize` and another was better under eol2, I'd strongly recommend we pick the first one, since it will likely be better for the majority of our users.

Reply:
Eh, valid enough